### ISSUE TITLE
webpack warns "request of a dependency is an expression"

### ISSUE THREAD

**Issue by city41** [Urigo/merge-graphql-schemas#137]
We are using merge-graphql-schemas in the browser. We don't use fileLoader, but since it is present in index.cjs.js, webpack warns when it encounters the dynamic require: 

```
./node_modules/merge-graphql-schemas/dist/index.cjs.js
3778:21-31 Critical dependency: the request of a dependency is an expression
```

Does it make sense to have a browser build of merge-graphql-schemas? It would be nice to drop fileLoader for the bundle size savings too. I can do a PR for that if you would like.

**Comment by cfnelson** [comment]
@city41 Personally, I am interested by the idea, however depending on the changes required to support this it might not be suitable. Main concern is if this would cause a higher maintenance overhead for the package. 

To save you any time before you write a PR it might be useful to know what modifications would be needed and your approach to the refactor?

Are you thinking of adding a `browser` field in the `package.json` and modifying the rollup build to produce a browser bundle that doesn't include the `fileLoader`?  Guessing we may want to also have tests for these different bundles. Documentation would need to be updated as well.

**Comment by city41** [comment]
Yeah that was the plan. I did this briefly and had a `index.browser.js` that exported mergeTypes and mergeResolvers but not fileLoader. But I ultimately ended up solving my problem by not using merge-graphql-schemas in the browser.

**Comment by ardatan** [comment]
Available in v1.6.0!
